jobserver/management/commands/create_project_from_workspace.py

```diff
@@ -0,0 +1,67 @@
+import sys
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from furl import furl
+
+from jobserver.models import Org, Project, Workspace
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("org")
+        parser.add_argument("--project")
+        parser.add_argument("workspace")
+
+    def _create_project(self, *, org, workspace):
+        """Create a Project for the given Workspace in the given Org"""
+
+        # remove common branch names
+        slug = (
+            workspace.name.removesuffix("-main")
+            .removesuffix("_main")
+            .removesuffix("-master")
+            .removesuffix("_master")
+        )
+
+        # generate a name from the slug
+        name = " ".join(word.capitalize() for word in slug.split("-"))
+
+        project = Project.objects.create(org=org, name=name, slug=slug)
```

Comment:
`get_or_create`?  Is it possible that there could be two workspaces for the same org with names e.g. "my_workspace" and "my_workspace_main", and this could try to create a project with a duplicate slug?  Should those two workspaces legitimately be assigned to the same project?

Reply:
It's possible but I think unlikely, and I'd rather this exploded when I used it so I knew there was an issue!  I'm passing the Org slug in via the CLI so if there was some clash it would at least go to the correct Org.